docs/releasing/publishing.md

```diff
@@ -1,15 +1,30 @@
 # Publishing GOV.UK Frontend
 
-1. Make sure you have cards in the "Backlog" column to:
-  - Update the GOV.UK Design System to use the latest release
-  - Update the GOV.UK Prototype Kit to use the latest release
-  - Update the GOV.UK Frontend Docs to use the latest release
+## Steps you must complete before you release a new version of GOV.UK Frontend
 
 2. Checkout **master** and pull latest changes.
+1. Raise new issues in the team GitHub repositories to:
+- Create release notes for the new release, for example, [#1986](https://github.com/alphagov/govuk-frontend/issues/1986)
```

Comment:
I wonder if this would look a little neater/be easier to scan:

`Create release notes for the new release (example card: [#1986](https://github.com/alphagov/govuk-frontend/issues/1986))`

so it outputs:

Create release notes for the new release (example card: [#1986](https://github.com/alphagov/govuk-frontend/issues/1986))

It also helps make it clearer that this is asking you to create the cards for the work, not do the work itself, which might be missed if you scan the instructions and jump straight to the list items.

Reply:
This sounds okay to me. What do you think @EoinShaughnessy?

Reply:
@hannalaakso Sounds good!

Reply:
Doc updated, thanks both 👍